doc/remotes.md

```diff
@@ -1,58 +1,81 @@
-LXD has four types of remotes:
+One use case for remote servers is to distribute images that can be used to create instances on local servers.
+See {ref}`remote-image-servers` for more information.
 
-- Static
-- Default
-- Global (per-system)
-- Local (per-user)
+You can also add a full LXD server as a remote server to your client.
+In this case, you can interact with the remote server in the same way as with your local daemon.
+For example, you can manage instances or update the server configuration on the remote server.
 
-### Static
+## Authentication
 
-Static remotes are:
+To be able to add a LXD server as a remote server, the server's API must be exposed, which means that its [`core.https_address`](server-options-core) server configuration option must be set.
 
-- `local` (default)
-- `ubuntu`
-- `ubuntu-daily`
+When adding the server, you must then authenticate with it using the chosen method for {ref}`authentication`.
 
-They are hardcoded and can't be modified by the user.
+See {ref}`server-expose` for more information.
 
-### Default
+## List configured remotes
 
-Automatically added on first use.
+% Include parts of the content from file [howto/images_remote.md](howto/images_remote.md)
+```{include} howto/images_remote.md
+   :start-after: <!-- Include start list remotes -->
+   :end-before: <!-- Include end list remotes -->
+```
+
+## Add a remote LXD server
+
+% Include parts of the content from file [howto/images_remote.md](howto/images_remote.md)
+```{include} howto/images_remote.md
+   :start-after: <!-- Include start add remotes -->
+   :end-before: <!-- Include end add remotes -->
+```
+
+## Select a default remote
+
+The LXD command-line client is pre-configured with the `local` remote, which is the local LXD daemon.
+
+To select a different remote as the default remote, enter the following command:
+
+    lxc remote switch <remote_name>
+
+To see which server is configured as the default remote, enter the following command:
 
-### Global (per-system)
+    lxc remote get-default
 
-By default the global configuration file is kept in either `/etc/lxd/config.yml`, or `/var/snap/lxd/common/global-conf/` for the snap version, or in `LXD_GLOBAL_CONF` if defined.
-The configuration file can be manually edited to add global remotes. Certificates for those remotes should be stored inside the `servercerts` directory (e.g. `/etc/lxd/servercerts/`) and match the remote name (e.g. `foo.crt`).
+## Configure a global remote
 
-An example configuration is below:
+You can configure remotes on a global, per-system basis.
+These remotes are available for every user of the LXD server for which you add the configuration.
+
+Users can override these system remotes (for example, by running `lxc remote rename` or `lxc remote set-url`), which results in the remote and its associated certificates being copied to the user configuration.
```

Comment:
Is this actually true? I could not get it to work. 
The `lxc remote rename` command didn't give an error but didn't rename anything, and the `lxc remote set-url` command gave me an error because I didn't have any certificates for the remote (so that error made sense).

Reply:
There's some amount of logic at least in the rename command to indeed copy the certificate across.
There very well could be bugs in that logic though ;)

Reply:
OK, I did some more testing now, and it seems to work as intended. The only issue I see is when I follow the example in the docs (create a global foo and bar with non-existing addresses) and don't provide any server certs. When I then try to rename foo, the command doesn't do anything (which makes sense, since it doesn't have a server cert) - but it also doesn't give an error.
Not sure if that's worth opening an issue?

Reply:
https://github.com/lxc/lxd/pull/11817